src/app/modules/users/components/users-list/users-list.component.ts

```diff
@@ -73,4 +105,18 @@ export class UsersListComponent implements OnInit, OnDestroy, AfterViewInit {
       ? this.store.dispatch(new RevokeRoleUser(userId, roleId))
       : this.store.dispatch(new GrantRoleUser(userId, roleId));
   }
+
+  switchGroup(userId: string, userGroups: string[], groupName: string) {
+    userGroups.includes(groupName)
+      ? this.store.dispatch(new RemoveUserFromGroup(userId, groupName))
+      : this.store.dispatch(new AddUserToGroup(userId, groupName));
+  }
+
+  isFeatureToggleActivated() {
+    return this.featureManagerService.isToggleEnabledForUser('switch-group').pipe(
+      map((enabled) => {
+        return enabled === true ? true : false;
```

Comment:
`enabled` is `boolean`, so this is unnecesary explicit comparation. Simple return `enabled`. e.g
```javascript
return this.featureManagerService.isToggleEnabledForUser('switch-group')
    .pipe(map((enabled: boolean) => enabled));
```

Reply:
it's an ES6 advantage 🔥